global_pose_initializer:
  ros__parameters:
    normal_estimation:
      search_radius: 2.0
    fpfh:
      search_radius: 4.0
    map_downsample_leaf_size: 0.5
    sensor_downsample_leaf_size: 0.5
    load_fpfh_feature: false
    fpfh_map_path: /tmp/fpfh_features.pcd
